components/Cases/CaseRecap/CaseRecap.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Spinner from 'components/Spinner/Spinner';
+import Summary from 'components/Summary/Summary';
+import { formatDate } from 'utils/date';
+import { useCaseByResident } from 'utils/api/cases';
+import * as form from 'data/forms';
+
+const CaseRecap = ({ personId, recordId }) => {
+  const { data: record, error: recordError } = useCaseByResident(
+    personId,
+    recordId
+  );
+
+  const recordData =
+    record && record.caseFormData && record.caseFormData.form_name_overall;
+  const fileData = form[recordData];
+
+  if (recordError) {
+    return <ErrorMessage />;
+  }
+  if (!record || !fileData) {
```

Comment:
I think if `fileData` doesn't exist (but `recordData` does) we should show an error, otherwise the spinner will just hang